lib/util/parser.js

```diff
@@ -12,7 +12,7 @@ const kAttributePattern =
   /\b(id|class)\s*(=\s*("([^"]+)"|'([^']+)'|(\S+)))?/ig;
 
 const reAttrPattern =
-  /\b([a-z][a-z0-9\-]*)\s*(=\s*("([^"]+)"|'([^']+)'|(\S+)))?/ig;
+  /\b([_a-z][_a-z0-9\-]*)\s*(=\s*("([^"]+)"|'([^']+)'|(\S+)))?/ig;
 
```

Comment:
Allows for attributes to start with underscores. This is useful for diffHTML internals.